spectral_cube/cube_utils.py

```diff
@@ -486,3 +487,51 @@ def convert_bunit(bunit):
             unit = None
 
     return unit
+
+
+def world_take_along_axis(cube, position_plane, axis):
+    '''
+    Apply a 2D plane of pixel positions along collapsed along a third axis
+    to the equivalent WCS coordinates. For example, this will convert `argmax`
```

Comment:
```suggestion
    Convert a 2D plane of pixel positions to the equivalent WCS coordinates.
    For example, this will convert `argmax`
```
The pixel positions can be obtained in any way, right?  `argmax` is a good example, but this code will work on any plane of pixel positions